docs/plugins/repository-hdfs.asciidoc

```diff
@@ -46,20 +46,60 @@ plugin folder and point `HADOOP_HOME` variable to it; this should minimize the a
 [[repository-hdfs-config]]
 ==== Configuration Properties
 
-Once installed, define the configuration for the `hdfs` repository through `elasticsearch.yml` or the
+Once installed, define the configuration for the `hdfs` repository through the
 {ref}/modules-snapshots.html[REST API]:
 
+[source,js]
+----
+PUT _snapshot/my_hdfs_repository
+{
+  "type": "hdfs",
+  "settings": {
+    "uri": "hdfs://namenode:8020/",
+    "path": "elasticsearch/respositories/my_hdfs_repository",
+    "conf.dfs.client.read.shortcircuit": "true"
+  }
+}
+----
+// CONSOLE
+// TEST[skip:we don't have hdfs set up while testing this]
+
+The following settings are supported:
+
+`uri`::
+
+    The uri address for hdfs. ex: "hdfs://<host>:<port>/". (Required)
+
+`path`::
+
+    The file path within the filesystem where data is stored/loaded. ex: "path/to/file". (Required)
+
+`load_defaults`::
+
+    Whether to load the default Hadoop configuration or not. (Enabled by default)
+
+`conf.<key>`::
+
+    Inlined configuration parameter to be added to Hadoop configuration. (Optional)
```

Comment:
Perhaps a link to documentation on what all the possible Hadoop config params are (if such a doc exists)?